tiledb/api/src/filter_list.rs

```diff
@@ -67,6 +68,12 @@ impl<'ctx> FilterList<'ctx> {
         }
     }
 
+    pub fn to_vec(&self) -> TileDBResult<Vec<Filter<'ctx>>> {
+        (0..self.get_num_filters()?)
+            .map(|f| self.get_filter(f))
```

Comment:
Huh. TIL that collect() returns the first error or the whole list. That's awesome!